api/v1beta2/network_types.go

```diff
@@ -501,48 +585,70 @@ func (s Subnets) FindEqual(spec *SubnetSpec) *SubnetSpec {
 }
 
 // FilterPrivate returns a slice containing all subnets marked as private.
-func (s Subnets) FilterPrivate() (res Subnets) {
+func (s Subnets) FilterPrivate() Subnets {
```

Comment:
Any special reason to remove the named return?

Reply:
that's makes sense. I was writing some test cases to get empty scenarios, but just realized it wont more required. I am reverting this change adding more unit cases to test empty values.